README.org

```diff
@@ -929,6 +929,7 @@ In addition, for more configurations, take a look at [[https://github.com/caisah
    - [[https://cestlaz.github.io/stories/emacs/][C'est la Z - Using Emacs Series]] - A series of beginner-friendly Emacs tutorials by Mike Zamansky (@zamansky).
    - [[https://blog.jft.rocks/emacs/emacs-from-scratch.html][Emacs from scratch]] - A guide to configure Emacs from scratch.
    - [[https://caiorss.github.io/Emacs-Elisp-Programming/][Emacs In a Box]] - A tutorial for emacs lisp and emacs customization.
+   - [[https://www.youtube.com/watch?v=rCMh7srOqvw&list=PLhXZp00uXBk4np17N39WvB80zgxlZfVwj][Emacs Doom Screencast]] - A video tutorial for emacs doom made by @zaiste
```

Comment:
Add a period to end the description.

emacs doom made by @zaiste.

Reply:
Good point. Done.